Environmental/new_crawlers/Weather forecast/anta_weather_forecast/consumer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+from kafka import KafkaConsumer
+from water_quality_cork.elastic import ElasticSearchClient
+
+from dotenv import load_dotenv
+
+
+"""
+In docker compose there is another exposed port (29092) for clients of Kafka
+
+"""
+
+ANTA_LAT = 36.963974
```

Comment:
should we have a `constants.py` file for variables that are shared betweeen producer/consumer?